plutus-core/changelog.d/20230621_104609_marty.stumpf_plt_6326_fix_callsite_inliner.md

```diff
@@ -0,0 +1,3 @@
+### Fixed
+
+- The callsite inliners used to wrap the function body of a function variable in `Dupable`, making it possible to independently rename the function body, creating free variables. This is now fixed.
```

Comment:
I don't understand this. Renaming cannot create free variables and it doesn't touch existing ones. Renaming must always produce an alpha-equal thing, so if that is not happening somewhere, it's a major bug.

Reply:
Did you see this [comment](https://github.com/input-output-hk/plutus/pull/5308#discussion_r1234313335)?

Reply:
Yes, but I didn't understand it either! :)
I think it's because it's been a long time since I last looked into the inliners and I don't understand them. I'm going to take a look and come back to you.

Reply:
Oh actually, you are right, as I found/explained elsewhere, it doesn't touch free variables. The renaming itself doesn't create the free variables. I'll re-phrase.